src/StripeTests/Services/FileUploads/StripeFileUploadServiceTest.cs

```diff
@@ -0,0 +1,85 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Reflection;
+    using System.Threading.Tasks;
+
+    using Stripe;
+    using Xunit;
+
+    public class StripeFileUploadServiceTest : BaseStripeTest
+    {
+        private const string FileUploadId = "file_123";
+        private const string FileName = "StripeTests.Resources.file_upload_logo.png";
+
+        private Stream stream;
+        private StripeFileUploadService service;
+        private StripeFileUploadListOptions listOptions;
+
+        public StripeFileUploadServiceTest()
+        {
+            this.service = new StripeFileUploadService();
+
+            this.stream = typeof(StripeFileUploadServiceTest).GetTypeInfo().Assembly.GetManifestResourceStream(FileName);
```

Comment:
I mimicked what was done in the Stripe.Net.Tests test suite, do you think it's the right solution?

Reply:
Yep, that seems fine.